swagger/swaggerDoc.yaml

```diff
@@ -0,0 +1,849 @@
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"name\": \"name1\",\n  \"url\": \"url1\"\n}"
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            $ref: "#/definitions/webcam_feed"
+  /api/webcam_feeds/{IntID}:
+    get:
+      produces:
+      - "application/json"
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            $ref: "#/definitions/webcam_feed"
+    delete:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    patch:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"name\": \"ok\",\n  \"url\": \"/foo.jpg\"\n}"
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            $ref: "#/definitions/webcam_feed"
+    parameters:
+    - name: "IntID"
+      in: "path"
+      required: true
+      type: "string"
+  /api/users:
+    post:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"email\": \"lera@runte.com\",\n  \"name\": \"Frank\"\
+            ,\n  \"password\": \"Password123\",\n  \"password_confirmation\": \"Password123\"\
+            \n}"
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+            example: "{\n  \"message\": \"Check your email!\"\n}"
+    delete:
+      parameters: []
+      responses:
+        200:
+          description: "Status 200"
+    patch:
+      consumes:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+        x-examples:
+          application/json: "{\n  \"new_password\": \"123456789\",\n  \"new_password_confirmation\"\
+            : \"123456789\",\n  \"password\": \"K5Ko82J340KaNl7s\"\n}"
+      responses:
+        200:
+          description: "Status 200"
+  /api/users/resend_verification:
+    post:
+      consumes: []
+      produces:
+      - "application/json"
+      parameters:
+      - name: "body"
+        in: "body"
+        required: true
+        schema:
+          type: "object"
+          example: "{\n  \"email\": \"oma_quitzon@gleasonmacejkovic.com\"\n}"
+      responses:
+        200:
+          description: "Status 200"
+          schema:
+            type: "object"
+          examples:
+            application/json: "{\n  \"user\": \"Check your email!\"\n}"
+  /api/users/verify/{token}:
+    put:
+      description: "why token in path-variable and in request-body?\npath-variable\
```

Comment:
This endpoint will be removed after the next release. It's there mostly for legacy reasons.